### PR TITLE
Lang pack

### LINKED ISSUE #114: Update of language packs would be good
It might also be a good idea to switch to [using keys instead the actual English strings](https://apluslms.github.io/contribute/styleguides/gettext/#msgid-format) as is done in the main A-plus repository. If that is done, it would make it easier to update translations.

In the main Aplus-repository, the word "exercise" has been replaced with assignment. It would be good for MOOC-grader to be consistent.

Also, in several places, the "trimmed" keyword is not used with translation blocks, that would be good to update.

### PR DESCRIPTION
**Pull request by etanttila**
# Description

**What?**

- Switch strings displayed to the user in the source code from the English strings to keys and move English strings to a localization file.
- Proofread and improve some texts, including "exercise" -> "assignment"
- Add translations to error pages (404 and 500) in case at some point grader starts supporting multilanguage browsing.

**Why?**

To make changing strings in the future easier, and to be similar to the main a-plus repository

**How?**
Read **What**

Fixes #114 

# Testing

**Remember to add or update unit tests for new features and changes.**

* How to [test your changes in A-plus](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations)
* How to [test accessibility](https://wiki.aalto.fi/display/EDIT/How+to+check+the+accessibility+of+pull+requests)

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [x] Manual testing.

I did some manual locally, but did not test that every single message is displayed correctly..

**Did you test the changes in**

- [ ] Chrome
- [x] Firefox
- [ ] This pull request cannot be tested in the browser.

**Think of what is affected by these changes and could become broken**

# Translation

- [x] Did you modify or add new strings in the user interface? ([Read about how to create translation](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations))

# Programming style

- [x] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?

# Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [ ] Aplus Manual.
- [ ] Other documentation (mention below which documentation).

If this is merged, the [gettext styleguide](https://apluslms.github.io/contribute/styleguides/gettext/#msgID-format) should be updated to mention that also this repository uses this styling.

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

File: locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,308 @@
+# English texts for the MOOC-Grader
+# Copyright (C) 2021 Ella Anttila
+# This file is distributed under the same license as the PACKAGE package.
```
Comment (Mankro) [inline locale/en/LC_MESSAGES/django.po:3]: The PACKAGE placeholder could be replaced with MOOC-Grader. I can change this myself.

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+# Ella Anttila <ella.anttila@aalto.fi>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (Mankro) [inline locale/en/LC_MESSAGES/django.po:8]: Again, PACKAGE is MOOC-Grader.

**Review by Mankro — APPROVED** [review]
Great work! Looks good.

**Comment by etanttila** [comment]
Definitely! I'll get on it! :)

**Comment by etanttila** [comment]
[Here's](https://github.com/apluslms/apluslms.github.io/pull/10) the PR for updating the documentation.

**Pull request merged**